### ISSUE TITLE
Color list [Feature]

### ISSUE THREAD

**Issue by ameintjes** [shaze/genesis#4]
It would be useful to be able to save the colour list for applying to other PCA datasets

This feature might be somewhere already

**Comment by shaze** [comment]
Yes, this may be OS dependant. On the Mac it seems to work, but I think its to do with the library we use.
